rest-server/templates/src/main/scala/com/actionml/templates/cb/CBEngine.scala

```diff
@@ -142,8 +142,8 @@ Query
 }
 */
 case class CBQuery(
-    user: String,
-    groupId: String)
```

Comment:
http://docs.scala-lang.org/style/declarations

double indentation of vertically justified parameter lists in class defs

In general do not change my style in my code and I will do the same for you unless it violates the Scala style guide. If it's important we should agree but generally if the style is allowed in the Scala style docs we allow the author and maintainer to use what they want in code they maintain.